JASP-Engine/JASP/R/auditCommonFunctions.R

```diff
@@ -1298,6 +1308,24 @@
 ################## Common functions for the Audit Risk Model ###################
 ################################################################################
 
+.auditCalculateDetectionRisk <- function(options){
+
+  inherentRisk <- 0
+  inherentRisk <- ifelse(options[["IR"]] == "High", yes = 1, no = inherentRisk)
```

Comment:
Wouldnt a `switch` be better?
```R
switch(options[["IR"]],
  High       = inherentRisk <- 1,
  Medium = inherentRisk <- 0.60,
  Low       = inherentRisk <- 0.50,
  Custom = inherentRisk <- options[["irCustom"]]
)
```

The code you wrote now looks like you are wishing your arm will get cut off ;)

Reply:
My poor arm...